docker/tests/integration/open-distro-docker-compose.yml

```diff
@@ -8,8 +8,8 @@ services:
   elasticsearch:
     image: "gcr.io/cicd-246518/es-open-distro:1.3.0"
     ports:
-      - "19200:9200"
-      - "19600:9600"
+      - "9200:9200"
```

Comment:
Why was the port changed? I think @wezell asked us to add a '1' at the beginning of the port number to avoid conflicts in case the default port was being used by another process

Reply:
Yeah this should be kept as "19200" and "19600". Probably local changes that shouldn't be here @victoralfaro-dotcms

Reply:
Bummer! That was the change I did to get the open distro image running locally.
Reverting it.